<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="fdp" name="org.jclouds.aliyun.ecs.AliyunEcsProviderTest" tests="4" failures="1" timestamp="7 Jan 2016 15:54:30 GMT" time="0.062" errors="3">
  <testcase name="testOfApiContains" time="0.015" classname="org.jclouds.aliyun.ecs.AliyunEcsProviderTest">
    <error type="java.lang.AssertionError" message="{id=aliyun-ecs, name=Aliyun Elastic Compute Service (ECS), api={id=aliyun-ecs, name=Aliyun Ecs API, views=[org.jclouds.aliyun.ecs.compute.AliyunEcsComputeServiceContext], endpointName=https endpoint, identityName=AccessKeyId, credentialName=Optional.of(Secret Key), documentation=http://download.cloud.com/releases/2.2.0/api_2.2.12/TOC_User.html, api=interface org.jclouds.aliyun.ecs.AliyunEcsApi}, endpoint=https://ecs.aliyuncs.com/, console=Optional.of(https://ecs.console.aliyun.com/#/), homepage=Optional.of(http://www.aliyun.com), linkedServices=[aliyun-ecs], iso3166Codes=[]} not found in []">
      <![CDATA[java.lang.AssertionError: {id=aliyun-ecs, name=Aliyun Elastic Compute Service (ECS), api={id=aliyun-ecs, name=Aliyun Ecs API, views=[org.jclouds.aliyun.ecs.compute.AliyunEcsComputeServiceContext], endpointName=https endpoint, identityName=AccessKeyId, credentialName=Optional.of(Secret Key), documentation=http://download.cloud.com/releases/2.2.0/api_2.2.12/TOC_User.html, api=interface org.jclouds.aliyun.ecs.AliyunEcsApi}, endpoint=https://ecs.aliyuncs.com/, console=Optional.of(https://ecs.console.aliyun.com/#/), homepage=Optional.of(http://www.aliyun.com), linkedServices=[aliyun-ecs], iso3166Codes=[]} not found in []
	at org.jclouds.providers.internal.BaseProviderMetadataTest.testOfApiContains(BaseProviderMetadataTest.java:61)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- testOfApiContains -->
  <testcase name="testTransformableToContains" time="0.000" classname="org.jclouds.aliyun.ecs.AliyunEcsProviderTest">
    <error type="java.lang.AssertionError" message="{id=aliyun-ecs, name=Aliyun Elastic Compute Service (ECS), api={id=aliyun-ecs, name=Aliyun Ecs API, views=[org.jclouds.aliyun.ecs.compute.AliyunEcsComputeServiceContext], endpointName=https endpoint, identityName=AccessKeyId, credentialName=Optional.of(Secret Key), documentation=http://download.cloud.com/releases/2.2.0/api_2.2.12/TOC_User.html, api=interface org.jclouds.aliyun.ecs.AliyunEcsApi}, endpoint=https://ecs.aliyuncs.com/, console=Optional.of(https://ecs.console.aliyun.com/#/), homepage=Optional.of(http://www.aliyun.com), linkedServices=[aliyun-ecs], iso3166Codes=[]} not found in [] for org.jclouds.aliyun.ecs.compute.AliyunEcsComputeServiceContext">
      <![CDATA[java.lang.AssertionError: {id=aliyun-ecs, name=Aliyun Elastic Compute Service (ECS), api={id=aliyun-ecs, name=Aliyun Ecs API, views=[org.jclouds.aliyun.ecs.compute.AliyunEcsComputeServiceContext], endpointName=https endpoint, identityName=AccessKeyId, credentialName=Optional.of(Secret Key), documentation=http://download.cloud.com/releases/2.2.0/api_2.2.12/TOC_User.html, api=interface org.jclouds.aliyun.ecs.AliyunEcsApi}, endpoint=https://ecs.aliyuncs.com/, console=Optional.of(https://ecs.console.aliyun.com/#/), homepage=Optional.of(http://www.aliyun.com), linkedServices=[aliyun-ecs], iso3166Codes=[]} not found in [] for org.jclouds.aliyun.ecs.compute.AliyunEcsComputeServiceContext
	at org.jclouds.providers.internal.BaseProviderMetadataTest.testTransformableToContains(BaseProviderMetadataTest.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- testTransformableToContains -->
  <testcase name="testAllContains" time="0.031" classname="org.jclouds.aliyun.ecs.AliyunEcsProviderTest">
    <error type="java.lang.AssertionError" message="{id=aliyun-ecs, name=Aliyun Elastic Compute Service (ECS), api={id=aliyun-ecs, name=Aliyun Ecs API, views=[org.jclouds.aliyun.ecs.compute.AliyunEcsComputeServiceContext], endpointName=https endpoint, identityName=AccessKeyId, credentialName=Optional.of(Secret Key), documentation=http://download.cloud.com/releases/2.2.0/api_2.2.12/TOC_User.html, api=interface org.jclouds.aliyun.ecs.AliyunEcsApi}, endpoint=https://ecs.aliyuncs.com/, console=Optional.of(https://ecs.console.aliyun.com/#/), homepage=Optional.of(http://www.aliyun.com), linkedServices=[aliyun-ecs], iso3166Codes=[]} not found in []">
      <![CDATA[java.lang.AssertionError: {id=aliyun-ecs, name=Aliyun Elastic Compute Service (ECS), api={id=aliyun-ecs, name=Aliyun Ecs API, views=[org.jclouds.aliyun.ecs.compute.AliyunEcsComputeServiceContext], endpointName=https endpoint, identityName=AccessKeyId, credentialName=Optional.of(Secret Key), documentation=http://download.cloud.com/releases/2.2.0/api_2.2.12/TOC_User.html, api=interface org.jclouds.aliyun.ecs.AliyunEcsApi}, endpoint=https://ecs.aliyuncs.com/, console=Optional.of(https://ecs.console.aliyun.com/#/), homepage=Optional.of(http://www.aliyun.com), linkedServices=[aliyun-ecs], iso3166Codes=[]} not found in []
	at org.jclouds.providers.internal.BaseProviderMetadataTest.testAllContains(BaseProviderMetadataTest.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- testAllContains -->
  <testcase name="testWithId" time="0.016" classname="org.jclouds.aliyun.ecs.AliyunEcsProviderTest">
    <failure type="java.util.NoSuchElementException">
      <![CDATA[java.util.NoSuchElementException
	at com.google.common.collect.AbstractIterator.next(AbstractIterator.java:154)
	at com.google.common.collect.Iterators.find(Iterators.java:712)
	at com.google.common.collect.Iterables.find(Iterables.java:642)
	at org.jclouds.providers.Providers.withId(Providers.java:99)
	at org.jclouds.providers.internal.BaseProviderMetadataTest.testWithId(BaseProviderMetadataTest.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- testWithId -->
</testsuite> <!-- org.jclouds.aliyun.ecs.AliyunEcsProviderTest -->
